pkg/domain/infra/abi/play.go

```diff
@@ -178,17 +178,10 @@ func (ic *ContainerEngine) playKubePod(ctx context.Context, podName string, podY
 		return nil, err
 	}
 
-	// check for name collision between pod and container
+	// Assert the pod has a name
 	if podName == "" {
 		return nil, errors.Errorf("pod does not have a name")
 	}
-	for _, n := range podYAML.Spec.Containers {
-		if n.Name == podName {
-			playKubePod.Logs = append(playKubePod.Logs,
-				fmt.Sprintf("a container exists with the same name (%q) as the pod in your YAML file; changing pod name to %s_pod\n", podName, podName))
-			podName = fmt.Sprintf("%s_pod", podName)
-		}
-	}
```

Comment:
I'm just wondering if it makes sense to keep this warning in play, although maybe changing the wording a bit, to let the user know of the collision so they could adjust afterwards if they wanted to.  I agree with dropping the auto renam though.

Reply:
The warning is missleasing. There never will be a container with the same name as the pod, since all container names in the pod will be prefixed with the pod name.